parsers/IL.py

```diff
@@ -88,19 +141,66 @@ def extract_price_date(soup):
 def fetch_production(
     zone_key="IL", session=None, target_datetime=None, logger=None
 ) -> dict:
-    if target_datetime:
-        raise NotImplementedError("This parser is not yet able to parse past dates")
 
-    data = fetch_all()
-    production = [float(item) for item in data]
-
-    # all mapped to unknown as there is no available breakdown
-    return {
-        "zoneKey": zone_key,
-        "datetime": arrow.now(TZ).datetime,
-        "production": {"unknown": production[0] + production[1]},
-        "source": IEC_URL,
-    }
+    data = {"zoneKey": zone_key, "datetime": arrow.now(TZ).datetime}
+
+    # fetch pv capacity from IRENA
+
+    try:
+        assert (
+            post(url=IRENA_CAPACITY_URL, json=IRENA_SOLAR_PV_QUERY).status_code == 200
```

Comment:
By doing this you're calling the URL twice. Let's put this in the `fetch_IRENA_capacity` base function